src/TimeService/timeservice-linux.h

```diff
@@ -0,0 +1,17 @@
+#include "timeservice-utils.h"
+
+#include <time.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <string.h>
+
+#define INIT_TIMERS()
+#define ADJUST_ENDIANESS(x)
+
+uint64_t get_clock()
```

Comment:
We are returning nanoseconds but function name is "get_clock", shouldn't it be "get_time" or something similar?

Reply:
OK